python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -169,4 +169,25 @@ def uniform_neighbor_sample(
         ddf = input_graph.unrenumber(ddf, "sources", preserve_order=True)
         ddf = input_graph.unrenumber(ddf, "destinations", preserve_order=True)
 
-    return ddf
+    # FIXME: this will convert to a dask_cudf.DataFrame even if arrays are to
+    # be returned so the unrenumbering step above can be used. A better
+    # approach would be to update the renumbering code to accept cupy arrays to
+    # renumber.
+    if (
+        hasattr(uniform_neighbor_sample, "_return_type")
+        and uniform_neighbor_sample._return_type == "arrays"
+    ):
+        uniform_neighbor_sample._return_type = ""
```

Comment:
Is this overwrite of `uniform_neighbor_sample._return_type` intentional ?

Reply:
Just saw the test, Can you add a comment here to explain why you over-write `_return_type` here